packages/php-wasm/logger/src/lib/collectors/collect-php-logs.ts

```diff
@@ -1,7 +1,4 @@
-import {
-	UniversalPHP,
-	PHPRequestErrorEvent,
-} from '@php-wasm/universal/src/lib/universal-php';
+import { UniversalPHP, PHPRequestErrorEvent } from '../../universal';
```

Comment:
These imports were causing circular dependency issues, so I added type declarations.

Reply:
I can always switch to `any` :sweat_smile: